### PR TITLE
Replaced alert with snackbar

### PR DESCRIPTION
**Pull request by smitthakkar96**

### CODE REVIEW

File: fstat/static/js/snackbar.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+$("body").prepend('<div id="snackbar"></div>')
```
Comment (nigelbabu) [inline fstat/static/js/snackbar.js]: Why not add this to the HTML directly?

Comment (smitthakkar96) [inline fstat/static/js/snackbar.js]: What if we use snackbar.js in multiple pages? In that case we will need to add that thing to all pages which I feel is inconvenient.

Comment (nigelbabu) [inline fstat/static/js/snackbar.js]: All templates are derived from base.html, isn't it? So you'll only add it on one template.

**Comment by nigelbabu** [comment]
Bah, merge conflicts.

**Pull request merged**